cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,22 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        while True:
```

Comment:
It is not a good practice to use an infinite loop (`while True`) without any condition to break it, even if you have a `break` statement inside the loop. This can lead to unexpected behavior if the condition inside the loop never becomes `True`. Consider adding a maximum number of attempts or a timeout value to prevent the loop from running indefinitely.